torch/csrc/jit/codegen/onednn/graph_helper.h

```diff
@@ -10,6 +10,9 @@ namespace jit {
 namespace fuser {
 namespace onednn {
 
+#define STRIDED_LAYOUT 0
+#define MKLDNN_LAYOUT 1
```

Comment:
Can you please rename it to `OPAQUE_LAYOUT` to signify that its strides-info is opaque to PyTorch? Thanks!
Even if the tensor between two oneDNN Graph partitions is strided (as it could be channels-last), we wrap it as an MKLDNN tensor just so that we can bypass the strides-check at partition execution time.